models/book.py

```diff
@@ -1,13 +1,15 @@
 from sqlalchemy import Column, Integer, Numeric, String, ForeignKey
 from sqlalchemy.orm import relationship
-from database import Base
+from upcase_booklist_app.database import Base
 
 
 class Book(Base):
 
     __tablename__ = "books"
 
-    id = Column(Integer(), primary_key=True)
+    id = Column(Integer, primary_key=True)
     title = Column(String(), nullable=False)
```

Comment:
One thing to consider is whether to declare an attribute `NOT NULLABLE` or not. It's really a matter of whether an attribute is essential for an object. In this case, I believe `title` is. To get this in your database, you just have to change `nullable=True` and run alembic autogenerate. Gives you another chance to practice migrations.

Look at the `NOT NULL` and `NULL` entries on the Postgres documentation:

https://www.postgresql.org/docs/10/sql-createtable.html

Reply:
I got mixed up on the above comment, but I think you'll want `NULLABLE` for `image_url` and `summary`.